### PR TITLE
Dev/1 1

### PR DESCRIPTION
**Pull request by justusdecker**

### CODE REVIEW

File: bin/auto/wf_fetch_audio.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                video_path = episodes[i].video_path
+                
+                prefix = f'{AUDIO_FOLDER}{i+1}_{self.lp_name}_'
+                ff = '.aac'
+                mic_track_path = f'{prefix}mic{ff}'
+                desktop_track_path = f'{prefix}desktop{ff}'
+                ffmpeg_stream_ammount = ffmpeg_run(FFMPEG_GET_STREAM_AMMOUNT,{'__IN__':video_path},True).strip()
+                reoc(not ffmpeg_stream_ammount.isdecimal(),'Something went wrong.\nThis error should not happen!\nFFPROBE HAS GIVEN BACK A NOT NUMERIC STRING')
+                
+                # We only do a soft edge case test here!
```
Comment (JustusTestSystem) [inline bin/auto/wf_fetch_audio.py:45]: Better edge case test

File: bin/auto/wf_fix_audio.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            super().user_workflow()
+        except AutomationError as AE:
+            xerr('Automation Error\n{AE}')
+        
+        pbm.reset_task()
+        
+        # After processing all episodes, we re-enabling the application's start button
+        # and calling the parent `user_workflow` to display the completion message.
+        # It does not matter whether the automation was completed or canceled.
+        for i in app.lpep_picker.get_ui():
```
Comment (JustusTestSystem) [inline bin/auto/wf_fix_audio.py:64]: put these in its own method

File: bin/auto/wf_generate_thumbnail.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            pbm.clean(len(rng))
+            
+            for ci,i in enumerate(rng): 
+                pbm.increment()
+                video_path = episodes[i].video_path
+                reoc(video_path is None,ERROR_013)
+                reoc(not isfile(video_path), ERROR_007)
+                
+                
+                p = f'{THUMBNAIL_FOLDER}{i+1}_{self.lp_name}_thumbnail.png'
```
Comment (JustusTestSystem) [inline bin/auto/wf_generate_thumbnail.py:47]: Rename p to e.g. path

File: bin/ui/nb_settings.py
Comment (JustusTestSystem) [inline bin/ui/nb_settings.py:1]: Naming things cant be that hard...

File: bin/ui/progress_bar_manager.py
Comment (JustusTestSystem) [inline bin/ui/progress_bar_manager.py:1]: add docstrings

File: testing/test_import_export.py
Comment (JustusTestSystem) [inline testing/test_import_export.py:1]: remove

File: testing/test_lpep.py
Comment (JustusTestSystem) [inline testing/test_lpep.py:1]: remove

File: testing/test_main.py
Comment (JustusTestSystem) [inline testing/test_main.py:1]: remove

File: testing/test_pbm.py
Comment (JustusTestSystem) [inline testing/test_pbm.py:1]: remove

**Review by JustusTestSystem — CHANGES_REQUESTED** [review]
